### PR TITLE
Initial status view for payments service

### LINKED ISSUE #7: Stand up the public payments service
This is to stand up the API service that will be publicly accessible by the payments UI and also for incoming webhooks. This will facilitate everything in the buy flow such as processing payments or events like cancelled subscriptions. For clarity, the Solitude service is private so we need a public service that the UI can talk to.

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/payments/issues/7

### CODE REVIEW

**Comment by kumar303** [comment]
@andymckay r?

I was running it with https://github.com/mozilla/payments-env/pull/2

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM mozillamarketplace/centos-mysql-mkt:0.2
+
+RUN yum install -y supervisor
+
+ENV IS_DOCKER 1
```
Comment (andymckay) [inline Dockerfile]: you can remove this line, we are trying to remove it from marketplace-env Dockerfile where you probably copied it from

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+RUN mkdir -p /pip/{cache,build}
+ADD requirements /pip/requirements
+WORKDIR /pip
+RUN pip install -b /pip/build --download-cache /pip/cache --no-deps -r /pip/requirements/dev.txt --find-links https://pyrepo.addons.mozilla.org/
```
Comment (andymckay) [inline Dockerfile]: we should consider using peep from the get go

File: manage.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+import os
+import sys
+
+if __name__ == "__main__":
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "payments_service.settings")
+
+    from django.core.management import execute_from_command_line
+
+    execute_from_command_line(sys.argv)
```
Comment (andymckay) [inline manage.py:10]: i love a clean manage.py, may it always stay this way

File: requirements/prod.txt [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Production requirements go here.
+# Be sure that you always specify an exact version such as:
+# foo==1.2.3 and you list all dependencies explicitly.
+
+curling==0.4
```
Comment (andymckay) [inline requirements/prod.txt]: you might want to grab 0.4.1 now

**Comment by andymckay** [comment]
r+ wc

File: Dockerfile (hunk 2 above)
Comment (kumar303) [inline Dockerfile]: I still don't know why peep wasn't a patch submitted to pip :disappointed: but I'll check it out

File: manage.py (hunk 3 above)
Comment (kumar303) [inline manage.py:10]: I didn't add anything for magic test settings. We might need it but maybe the docker env will make things predictable enough.

**Pull request merged**